app/assets/stylesheets/embed.scss

```diff
@@ -176,6 +176,20 @@ div.lightbox-wrapper {
   margin-bottom: 20px;
 }
 
+.new-topic-btn {
```

Comment:
This should match the default Discourse styles for primary buttons (blue background, + icon)